quantus/helpers/model_interface.py

```diff
@@ -43,3 +43,16 @@ def get_random_layer_generator(self):
         Set order to independent for independent randomization.
         """
         raise NotImplementedError
+
```

Comment:
Maybe refactor names to `get_hidden_layers_representations` or `get_hidden_layers_activations`?

Reply:
done